### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Cleaning up

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -220,9 +199,7 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
     ynh_script_progression --message="Upgrading Leed with cURL..." --weight=4
     # Set the app as temporarily public for curl call
-    ynh_app_setting_set --app=$app --key=unprotected_uris --value="/"
-    # Regen SSOwat configuration
-    yunohost app ssowatconf
+    ynh_permission_update --permission="main" --add="visitors"
```
Comment (alexAubin) [inline scripts/upgrade:202]: Eeeeh it looks like we should check if visitors is allowed, and if not, only *temporarily* add it (so this should be undone after the local curl)

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade (hunk 1 above)
Comment (alexAubin) [reply scripts/upgrade:202]: (Nevermind it's actually handled through is_public later, which should migrated at some point but in the meantime that's okay I guess)

**Review by alexAubin — APPROVED** [review]

**Pull request merged**